dashboard/config/linux/upstream-apparmor-kasan.config

```diff
@@ -1007,6 +1022,7 @@ CONFIG_SWAP=y
 CONFIG_ZSWAP=y
 CONFIG_ZSWAP_DEFAULT_ON=y
 # CONFIG_ZSWAP_EXCLUSIVE_LOADS_DEFAULT_ON is not set
+# CONFIG_ZSWAP_SHRINKER_DEFAULT_ON is not set
```

Comment:
Will this give us more coverage for zswap?

Reply:
Let's try.